qcodes/tests/dataset/test_measurement_context_manager.py

```diff
@@ -1127,15 +1147,9 @@ def test_datasaver_parameter_with_setpoints_reg_but_missing(
 
     param.setpoints = old_setpoints
     with meas.run() as datasaver:
-        with pytest.raises(ValueError, match=r"Can not add this result; "
-                                             r"missing setpoint values for "
-                                             r"dummy_channel_inst_ChanA_dummy_"
-                                             r"parameter_with_setpoints: "
-                                             r"\['dummy_channel_inst_ChanA_"
-                                             r"dummy_sp_axis', 'someparam'\]"
-                                             r". Values only given for \["
-                                             r"'dummy_channel_inst_ChanA_dummy_parameter_with_setpoints', "
-                                             r"'dummy_channel_inst_ChanA_dummy_sp_axis'\]"):
+        match = re.escape('Can not add result, some required parameters '
+                          'are missing.')
```

Comment:
would it be possible to know which "some" required parameters are missing as it was before this PR?